src/main/java/codeu/controller/AdminServlet.java

```diff
@@ -77,6 +77,7 @@ void setConversationStore(ConversationStore conversationStore) {
   }
 
   /**
+>>>>>>> ea7ba240a1be24824241b46686ed62675421fde2
```

Comment:
This line points out where part of a merge conflict starts / where it ends.

Once you've resolved the conflict, it should be removed.

Reply:
Done.